### PR TITLE
Gobierto Visualizations / Costs: refactor and update data

### PR DESCRIPTION
**Pull request by jorgeatgu**
Closes https://github.com/PopulateTools/issues/issues/1335

## :v: What does this PR do?

- Refactor the code to remove references to the year 2019.
- Add new text about covid19
- Update 2019 and 2020 population data.

## :mag: How should this be manually tested?

[Staging](https://mataro.gobify.net/visualizaciones/costes/)

### CODE REVIEW

File: app/javascript/gobierto_visualizations/modules/costs_controller.js [hunk 1]
```diff
@@ -150,23 +150,28 @@ export class CostsController {
...
+      d["population"] = getPopulation(d["any_"]);
       for (let amounts = 0; amounts < amountStrings.length; amounts++) {
         d[amountStrings[amounts]] = convertStringToNumbers(
           d[amountStrings[amounts]]
         );
       }
     }
 
-    let yearsCosts = [...new Set(rawData.map(item => item.any_))];
+    function getPopulation(year) {
```
Comment (Crashillo) [inline app/javascript/gobierto_visualizations/modules/costs_controller.js]: This function is totally worthless:
1. There is only one use (L160)
2. It's running over and over through the yearsCosts array
3. yearsCosts array is defined outside the function

Why not this? The current code is quite hard to understand
```
const ix = yearsCosts.findIndex(x => x === d["any_"])
d.population = population[ix]
```

**Review by Crashillo — CHANGES_REQUESTED** [review]

File: app/javascript/gobierto_visualizations/modules/costs_controller.js (hunk 1 above)
Comment (jorgeatgu) [reply app/javascript/gobierto_visualizations/modules/costs_controller.js]: Much better, done.

**Comment by furilo** [comment]
What's pending to merge this?

**Comment by jorgeatgu** [comment]
> What's pending to merge this?

Last Friday, they changed some data, so we are pending their review.

**Comment by ferblape** [comment]
Careful, the test that it's failing is caused by the text you updated, please review https://app.circleci.com/pipelines/github/PopulateTools/gobierto/4546/workflows/4cb58281-9e50-497c-bfe6-c690a0fb848b/jobs/39397

**Comment by jorgeatgu** [comment]
> Careful, the test that it's failing is caused by the text you updated, please review https://app.circleci.com/pipelines/github/PopulateTools/gobierto/4546/workflows/4cb58281-9e50-497c-bfe6-c690a0fb848b/jobs/39397

Done!

**Pull request merged**